pennylane_lightning/lightning_qubit.py

```diff
@@ -354,40 +406,61 @@ def apply_lightning(self, state, operations):
         Returns:
             array[complex]: the output state tensor
         """
-        state_vector = np.ravel(state)
+        batch_size_op = self.operations_batch_size(operations)
 
-        if self.use_csingle:
-            # use_csingle
-            sim = StateVectorC64(state_vector)
+        dim = 1 << self.num_wires
+        batch_size_state = self._get_batch_size(state, (2,) * self.num_wires, dim)
```

Comment:
Is it possible to change 'batch_size_state' to 'batch_size' to make it consistent with previous text (like #L393)?

Reply:
My intention with the suffix was to have it on the same foot as `batch_size_op`.
But I can change it. Not a problem, at all.

Reply:
Just take the one better in your mind.